notifications/endpoint.go

```diff
@@ -63,7 +65,11 @@ func NewEndpoint(name, url string, config EndpointConfig) *Endpoint {
 		endpoint.Transport, endpoint.metrics.httpStatusListener())
 	endpoint.Sink = newRetryingSink(endpoint.Sink, endpoint.Threshold, endpoint.Backoff)
 	endpoint.Sink = newEventQueue(endpoint.Sink, endpoint.metrics.eventQueueListener())
-	endpoint.Sink = newIgnoredMediaTypesSink(endpoint.Sink, config.IgnoredMediaTypes)
+	if len(config.IgnoredMediaTypes) <= 0 {
+		endpoint.Sink = newIgnoredSink(endpoint.Sink, config.Ignore.MediaTypes, config.Ignore.Actions)
+	} else {
+		endpoint.Sink = newIgnoredSink(endpoint.Sink, config.IgnoredMediaTypes, config.Ignore.Actions)
+	}
```

Comment:
One other nit: If both `config.Ignore.MediaTypes` and `config.IgnoredMediaTypes` are set, I'd expect both to be honored, so I'd do something like this here:
```go
mediaTypes := append(config.Ignore.MediaTypes, config.IgnoredMediaTypes...)
endpoint.Sink = newIgnoredSink(endpoint.Sink, mediaTypes, config.Ignore.Actions)
```

Reply:
It's a good idea.